openreview/conference/invitation.py

```diff
@@ -730,30 +731,39 @@ def __init__(self, conference, note):
         readers = review_stage.get_readers(conference, note.number)
         nonreaders = review_stage.get_nonreaders(conference, note.number)
 
+        reply = {
+            'forum': note.id,
+            'replyto': note.id,
+            'readers': {
+                'description': 'Select all user groups that should be able to read this comment.',
+                'values': readers
+            },
+            'nonreaders': {
+                'values': nonreaders
+            },
+            'writers': {
+                'values-regex': signature_regex,
+                'description': 'How your identity will be displayed.'
+            },
+            'signatures': {
+                'values-regex': signature_regex,
+                'description': 'How your identity will be displayed.'
+            }
+        }
+
+        has_copies = [r for r in readers if r.startswith('{') and r.endswith('}')]
+        if has_copies:
+            reply['readers'] = {
+                'description': 'Select all user groups that should be able to read this comment.',
+                'values-copied': readers
```

Comment:
Should this be `'values-copied': has_copies` instead ? 

Also, I would like to rename has_copies to something else, maybe copy_values. `has_copies` seems to imply a boolean.

Reply:
`has_copies` is a boolean that checks if there is at least one element in the array between curly braces.